faster_whisper/utils.py

```diff
@@ -76,6 +77,7 @@ def download_model(
 
     allow_patterns = [
         "config.json",
+        "preprocessor_config.json",
```

Comment:
@Oscaarjs , hello.  We have generated the Systran Whisper large-v3 conversion model with new file preprocessor_config.json in the HF-to-ct2 script, could you please update this info also in README.md file of your PR.
Example: ct2-transformers-converter --model openai/whisper-large-v3 --output_dir whisper-large-v2-ct2 \
    --copy_files tokenizer.json **preprocessor_config.json** --quantization float16

Reply:
@trungkienbkhn Ah yes! Did the changes, please check if I made them correctly

Reply:
Sounds good to me. Hopefully there will be some benchmark tests for the faster whisper large-v3 soon.